specification/communication/data-plane/PhoneNumbers/preview/2023-10-01-preview/phonenumbers.json

```diff
@@ -427,6 +427,59 @@
         }
       }
     },
+    "/availablePhoneNumbers/countries/{countryCode}/:browse": {
+      "post": {
+        "tags": [
+          "Browse, reserve and purchase available phone numbers"
+        ],
+        "summary": "Searches for available phone numbers to purchase. Note that this does not reserves the numbers in the response.",
+        "operationId": "PhoneNumbers_BrowseAvailableNumbers",
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "path",
+            "name": "countryCode",
+            "description": "The ISO 3166-2 country code, e.g. US.",
```

Comment:
Is this truly upper case, or are you case-insensitive? HTTP recommends case-sensitive paths, and I'm used to seeing e.g., "us" not "US", hence the question.

Reply:
This is case insensitive, which is the same behavior for our GA APIs as well. That said, I do think it makes sense for both "US" and "us" to be valid ways to identify a country.